### DISCUSSION TITLE
writing/2024/09/21/chase-bank-small-business-nightmare/

### DISCUSSION THREAD

**Discussion by giscus** [jxnl/blog#89] [General]
# writing/2024/09/21/chase-bank-small-business-nightmare/

A cautionary tale of a business owner's nightmare with Chase Bank, revealing crucial banking lessons for entrepreneurs.

https://jxnl.co/writing/2024/09/21/chase-bank-small-business-nightmare/

**Comment by AgenP** [comment]
Thank you for sharing this story and your lessons learned. Opening up a business bank account rn and definitely going to operate with these lessons top of mind
